admin/services/check-form.service.js

```diff
@@ -48,10 +48,9 @@ const checkFormService = {
 
   /**
    * Extract the questions from the check-form, and add an `order` property.
-   * @param checkForm
+   * @param questions
    */
-  prepareQuestionData: function (checkForm) {
-    const questions = JSON.parse(checkForm.formData)
```

Comment:
hard to read the flow in diff view, but i take it you are parsing the raw JSON string contained in `formData` elsewhere?

Reply:
Yes, it's parsed in `checkStartService.pupilLogin` before it is sent back to the controller

Reply:
👍